db/seed_data/proceeding_types.yml

```diff
@@ -4,60 +4,73 @@
 # - meaning
 # - description
 # - method to call to get the matter_type id
+# - additional_search_terms
 #
 - - DA006
   - Extend, variation or discharge - Part IV
   - to be represented on an application to extend, vary or discharge an order under
     Part IV Family Law Act 1996
   - domestic_abuse_id
+  -
 - - DA007
   - Forced marriage protection order
   - to be represented on an application for a forced marriage protection order
   - domestic_abuse_id
+  -
 - - DA003
   - Harassment - injunction
   - to be represented in an action for an injunction under section 3 Protection from
     Harassment Act 1997.
   - domestic_abuse_id
+  -
 - - DA001
   - Inherent jurisdiction high court injunction
   - to be represented on an application for an injunction, order or declaration under
     the inherent jurisdiction of the court.
   - domestic_abuse_id
+  -
 - - DA004
   - Non-molestation order
   - to be represented on an application for a non-molestation order.
   - domestic_abuse_id
+  - harassment,injunction
```

Comment:
for uniformity should there be a newline here between DA004 and DA005?

Reply:
There isn't a newline between the other entries, there's an empty yaml line because there are no additional_search_terms on the other entries